docs/latency_config.md

```diff
@@ -0,0 +1,35 @@
+## JSON Configuration
+
+Latency configuration (in terms of pipeline stages) for each computational operation used in a DFCxx kernel is provided via a JSON file.
```

Comment:
There is no information about pipeline stages in this document, so the refinement looks like "hanging in the air" fact.

Also passive voice is used, it makes sentence complicated. It would be more clear to start like "Latency configuration is a JSON-based file describing characteristics of computational operations for the specific DFCxx kernel."